experimental/plugins/postgres_storage/src/postgres_storage.rs

```diff
@@ -617,42 +633,81 @@ impl WalletStrategy for DatabasePerWalletStrategy {
     }
 }
 
+
+// determine additional query parameters based on wallet strategy
+fn get_multi_database_name(config: &PostgresConfig) -> &str {
+    // look to see if there is a specified db to use.  If not, use the default name
+    let name: &str = match config.database_name {
+        Some(ref database_name) => {
+            if database_name.is_empty() {
+                _WALLETS_DB
+            } else {
+                database_name
+            }
+        },
+        None => _WALLETS_DB,
+    };
+
+    return name;
+}
+
 impl WalletStrategy for MultiWalletSingleTableStrategy {
     // initialize storage based on wallet storage strategy
     fn init_storage(&self, config: &PostgresConfig, credentials: &PostgresCredentials) -> Result<(), WalletStorageError> {
+        match env_logger::try_init() {
```

Comment:
Why did you add logger initialization `init_storage` functions?  The logger will already be initialized on library loading here:
 https://github.com/hyperledger/indy-sdk/blob/5d3587a183399a14da680bb4e7e4c8540d1f6dea/experimental/plugins/postgres_storage/src/lib.rs#L59
Either Libindy callbacks or default env_logger will be set.

Reply:
removed